consensus-types/interfaces/beacon_block.go

```diff
@@ -73,7 +73,6 @@ type ReadOnlyBeaconBlockBody interface {
 
 type ROBlockBodyElectra interface {
 	ReadOnlyBeaconBlockBody
-	Consolidations() []*ethpb.SignedConsolidation
 }
```

Comment:
This interface becomes useless now, it can be deleted and simply ReadOnlyBeaconBlockBody